public/wp-content/themes/ColoredCow/landing.php

```diff
@@ -5,9 +5,88 @@
     get_header();
 ?>
 
-<?php
-?>
- 
+<body>
+	<div class="container">
+		<div class="row">
+			<div class="col col-sm-12 col-md- col-lg-6">
+				<h2 class="about">Soiree</h2>
+				<br>
+			  	<p class="lead">ColoredCow celebrates every first Saturday of the month with family and friends. This custom has been started to take a little time off from work and enjoy some moments in life. we believe in sharing moments and learning with each other. Come and join us over music, food, drinks and some moments full of laughter and joy.</p>
+				<hr>
+				<center>
+			  		<div class="col-md-8 col-xs-12">
+						<h5 class="request">Wanna join the party?</h5>
+						<br>
+						<button type="button" class="btn btn-outline-warning btn-lg btn-block" data-toggle="modal" data-target="#exampleModal" data-whatever="@mdo">Request Invite</button>		
+					</div>
+				</center>		
+			</div>	
+		</div>
+		<div class="container">
+            <div class="modal fade" id="exampleModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+                  <div class="modal-dialog" role="document">
+                    <div class="modal-content">
+                          <div class="modal-header">
+                            <h3 class="modal-title" id="exampleModalLabel">New Request</h3>
+                            <button type="button" class="close" data-dismiss="modal" aria-label="Close" id="close"><span aria-hidden="true">&times;</span></button>
+                          </div>
+                         <form id="request_form">
+                              <div class="modal-body">
+                                  <p id="msg2"></p>
+                                <input type="hidden" class="form-control" name="request_id" id="request_id">
+                                <div class="form-group">
+                                    <label for="recipient-name" class="form-control-label">
+                                        <h6>Your Name:</h6>
+                                    </label>
+                                    
+                                        <input type="text" class="form-control" placeholder="" name="request_name" id="request_name" maxlength="30" required>
```

Comment:
Remove the placeholder attribute if not using. This is only making your HTML statement longer and difficult to read.